{"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3071", "repository_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT", "labels_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3071/labels{/name}", "comments_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3071/comments", "events_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3071/events", "html_url": "https://github.com/Significant-Gravitas/Auto-GPT/issues/3071", "id": 1680242699, "node_id": "I_kwDOJKSTjM5kJngL", "number": 3071, "title": "start autogpt from master branch with any plugin seem doesent work", "user": {"login": "SimDamDev", "id": 79875090, "node_id": "MDQ6VXNlcjc5ODc1MDkw", "avatar_url": "https://avatars.githubusercontent.com/u/79875090?v=4", "gravatar_id": "", "url": "https://api.github.com/users/SimDamDev", "html_url": "https://github.com/SimDamDev", "followers_url": "https://api.github.com/users/SimDamDev/followers", "following_url": "https://api.github.com/users/SimDamDev/following{/other_user}", "gists_url": "https://api.github.com/users/SimDamDev/gists{/gist_id}", "starred_url": "https://api.github.com/users/SimDamDev/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/SimDamDev/subscriptions", "organizations_url": "https://api.github.com/users/SimDamDev/orgs", "repos_url": "https://api.github.com/users/SimDamDev/repos", "events_url": "https://api.github.com/users/SimDamDev/events{/privacy}", "received_events_url": "https://api.github.com/users/SimDamDev/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "open", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 1, "created_at": "2023-04-24T00:35:06Z", "updated_at": "2023-04-24T00:36:33Z", "closed_at": null, "author_association": "NONE", "active_lock_reason": null, "body": "### \u26a0\ufe0f Search for existing issues first \u26a0\ufe0f\n\n- [X] I have searched the existing issues, and there is no existing issue for my problem\n\n### Which Operating System are you using?\n\nWindows\n\n### Which version of Auto-GPT are you using?\n\nMaster (branch)\n\n### GPT-3 or GPT-4?\n\nGPT-4\n\n### Steps to reproduce \ud83d\udd79\n\nhave any plugin in plugin folder\r\nstart autogpt\n\n### Current behavior \ud83d\ude2f\n\n_No response_\n\n### Expected behavior \ud83e\udd14\n\n_No response_\n\n### Your prompt \ud83d\udcdd\n\n```yaml\r\n# Paste your prompt here\r\n```\r\n\n\n### Your Logs \ud83d\udcd2\n\n python -m autogpt\r\nNEWS:  Welcome to Auto-GPT! We'll keep you informed of the latest news and features by printing messages here. If you don't wish to see this message, you can run Auto-GPT with the --skip-news flag\r\nWARNING:  You are running on `master` branch - this is not a supported branch.\r\nWARNING: Plugin AutoGPTDollyPlugin found. But not in the allowlist... Load? (y/n): y\r\n\r\nPlugins found: 1\r\n--------------------\r\nAuto-GPT-Dolly-Plugin: 0.1.0 - This plugin adds a 'create_clone' command that lets Auto-GPT build an army of powerful minions.\r\nWelcome back!  Would you like me to return to being LMMGPT?\r\nContinue with the last settings?\r\nName:  LMMGPT\r\nRole:  an AI assistant that guides users through the process of adding a local LMM model to AutoGPT, providing step-by-step instructions and troubleshooting support.\r\nGoals: ['Provide clear and concise instructions for installing and setting up the necessary software and dependencies required for adding a local LMM model to AutoGPT.', 'Guide users through the process of preparing their data and fine-tuning the LMM model to ensure optimal performance and accuracy.', 'Offer troubleshooting support and solutions for common issues that may arise during the process of adding a local LMM model to AutoGPT.', 'Provide ongoing support and updates to ensure that the LMM model remains up-to-date and continues to perform optimally.', 'Ensure that users have a thorough understanding of the process and are able to successfully add a local LMM model to AutoGPT on their own.']\r\nAPI Budget: infinite\r\nContinue (y/n): y\r\nLMMGPT  has been created with the following details:\r\nName:  LMMGPT\r\nRole:  an AI assistant that guides users through the process of adding a local LMM model to AutoGPT, providing step-by-step instructions and troubleshooting support.\r\nGoals:\r\n-  Provide clear and concise instructions for installing and setting up the necessary software and dependencies required\r\n for adding a local LMM model to AutoGPT.\r\n-  Guide users through the process of preparing their data and fine-tuning the LMM model to ensure optimal performance and accuracy.\r\n-  Offer troubleshooting support and solutions for common issues that may arise during the process of adding a local LMM model to AutoGPT.\r\n-  Provide ongoing support and updates to ensure that the LMM model remains up-to-date and continues to perform optimally.\r\n-  Ensure that users have a thorough understanding of the process and are able to successfully add a local LMM model to AutoGPT on their own.\r\nUsing memory of type:  PineconeMemory\r\nUsing Browser:  chrome\r\nTraceback (most recent call last):\r\n  File \"<frozen runpy>\", line 198, in _run_module_as_main\r\n  File \"<frozen runpy>\", line 88, in _run_code\r\n  File \"C:\\Users\\simon\\OneDrive\\Documents\\GitHub\\Auto-GPT\\autogpt\\__main__.py\", line 5, in <module>\r\n    autogpt.cli.main()\r\n  File \"C:\\Users\\simon\\AppData\\Roaming\\Python\\Python311\\site-packages\\click\\core.py\", line 1130, in __call__\r\n    return self.main(*args, **kwargs)\r\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"C:\\Users\\simon\\AppData\\Roaming\\Python\\Python311\\site-packages\\click\\core.py\", line 1055, in main\r\n    rv = self.invoke(ctx)\r\n         ^^^^^^^^^^^^^^^^\r\n  File \"C:\\Users\\simon\\AppData\\Roaming\\Python\\Python311\\site-packages\\click\\core.py\", line 1635, in invoke\r\n    rv = super().invoke(ctx)\r\n         ^^^^^^^^^^^^^^^^^^^\r\n  File \"C:\\Users\\simon\\AppData\\Roaming\\Python\\Python311\\site-packages\\click\\core.py\", line 1404, in invoke\r\n    return ctx.invoke(self.callback, **ctx.params)\r\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"C:\\Users\\simon\\AppData\\Roaming\\Python\\Python311\\site-packages\\click\\core.py\", line 760, in invoke\r\n    return __callback(*args, **kwargs)\r\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"C:\\Users\\simon\\AppData\\Roaming\\Python\\Python311\\site-packages\\click\\decorators.py\", line 26, in new_func\r\n    return f(get_current_context(), *args, **kwargs)\r\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"C:\\Users\\simon\\OneDrive\\Documents\\GitHub\\Auto-GPT\\autogpt\\cli.py\", line 174, in main\r\n    system_prompt = ai_config.construct_full_prompt()\r\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"C:\\Users\\simon\\OneDrive\\Documents\\GitHub\\Auto-GPT\\autogpt\\config\\ai_config.py\", line 142, in construct_full_prompt\r\n    prompt_generator = plugin.post_prompt(prompt_generator)\r\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\r\n  File \"plugins\\AutoGPTDolly.zip\\src\\auto_gpt_plugin_template\\__init__.py\", line 48, in post_prompt\r\n  File \"plugins\\AutoGPTDolly.zip\\src\\auto_gpt_plugin_template\\dolly.py\", line 4, in <module>\r\nImportError: cannot import name 'path_in_workspace' from 'autogpt.workspace' (C:\\Users\\simon\\OneDrive\\Documents\\GitHub\\Auto-GPT\\autogpt\\workspace\\__init__.py)", "reactions": {"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3071/reactions", "total_count": 0, "+1": 0, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 0}, "timeline_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3071/timeline", "performed_via_github_app": null, "state_reason": null}